engine.py

```
@@ -1,81 +1,86 @@
+            tempfile = os.getcwd() + "/audio/GLaDOS-tts-temp-output-" + key + ".wav"
+
+            # If the line isn't too long, store in cache
+
+            if len(line) < 200 and CACHE:
+                shutil.move(tempfile, file)
+            else:
+                return send_file(tempfile)
+                os.remove(tempfile)
+            return send_file(file)
+        else:
+            return "TTS Engine Failed"
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)